api/src/pcapi/routes/e2e/routes/sandbox_data.py

```diff
@@ -76,6 +76,30 @@ def setup_create_venue() -> None:
     offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
 
 
+@app.route("/e2e/pro/update-venue", methods=["GET"])
+@spectree_serialize(on_success_status=204)
+def setup_update_venue() -> None:
+    clean_all_database()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ ‏si on clean all databases à chaque route, c'est pas problématique ? 
Peut être pas, mais si on appelle 2 routes à la suite, si

Reply:
On appelle jamais 2 route à la suite pour ce qu'on pour l'instant, chaque test appelle une route, setup ses données, le test suivant vide la db setup ses données etc...